server/live-loan-router.js

```diff
@@ -139,6 +142,27 @@ module.exports = function liveLoanRouter(cache) {
 		});
 	});
 
+	// User URL Router FLSS
+	router.use('/flss/u/:id(\\d{0,})/url/:offset(\\d{0,})', async (req, res) => {
+		await tracer.trace('live-loan.user.redirectToUrl', { resource: req.path }, async () => {
```

Comment:
I know we aren't actively using the tracers, but is it an issue that this tracer matches the existing `/u` tracing?

Reply:
Maybe we change to `live-loan.flss`?

Reply:
Also, https://kiva.atlassian.net/browse/MP-45?focusedCommentId=352345

Reply:
Yup I saw that comment. I just didn't want us to send in data that might confuse existing tracings.

Reply:
For now I think we can just change these to `live-loan.flss`